### PR TITLE
add warning when changing "Sharing" setting of someone else's export

### PR DESCRIPTION
**Pull request by nickpell**
screen shot 2018-11-05 at 11 43 59 am

Feature flag: https://www.commcarehq.org/hq/flags/edit/export_ownership/

### CODE REVIEW

File: corehq/apps/export/templates/export/customize_export_new.html [hunk 1]
```diff
@@ -167,6 +167,26 @@
                             value: sharing
                         ">
                     </select>
+                    {% if has_other_owner %}
+                    <div data-bind="visible: sharing() != initialSharing()" class="alert alert-warning">
```
Comment (orangejenny) [inline corehq/apps/export/templates/export/customize_export_new.html]: Can you replace `.alert.alert-warning` with `.help-block` and use `.has-warning` on the `.form-group` container when this warning is visible? Alert classes aren't meant to be used as form field feedback - the alert border right under to the select box without any spacing looks weird.

**Review by orangejenny — CHANGES_REQUESTED** [review]
CSS change, see comment

File: corehq/apps/export/templates/export/customize_export_new.html (hunk 1 above)
Comment (nickpell) [reply corehq/apps/export/templates/export/customize_export_new.html]: updated

File: corehq/apps/export/static/export/js/models.js [hunk 2]
```diff
@@ -56,6 +56,9 @@ hqDefine('export/js/models', function () {
             return format;
         });
 
+        self.initialSharing = ko.observable(instanceJSON.sharing);
```
Comment (orangejenny) [inline corehq/apps/export/static/export/js/models.js]: Is it deliberate that these are observables? Doesn't look like their values will ever change.

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/export/static/export/js/models.js (hunk 2 above)
Comment (nickpell) [reply corehq/apps/export/static/export/js/models.js]: Good point.  Will follow up with another PR (I want to get this one in for deploy)

Comment (nickpell) [reply corehq/apps/export/static/export/js/models.js]: https://github.com/dimagi/commcare-hq/pull/22285

**Pull request merged**